src/main/java/core/basesyntax/service/impl/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.CsvFileReader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReaderImpl implements CsvFileReader {
+    @Override
+    public List<TransactionDto> readDate(String fileName) {
+        List<TransactionDto> transactionDtoList = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String row;
+            while ((row = bufferedReader.readLine()) != null) {
+                String[] split = row.split(",");
+                ParseTransactionImpl parseTransaction = new ParseTransactionImpl();
+                transactionDtoList.add(parseTransaction.parseDateFromFile(split));
```

Comment:
let's either make `readDate(String fileName)` return `List<String[]>`, receive it in main() and pass to `parseTransaction.parseDateFromFile()` method

or make `parseDateFromFile(String[] transaction)` method private in this class